job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -384,7 +404,12 @@ class WebApi(system: ActorSystem,
                 val resultFuture = jobInfo ? GetJobResult(jobId)
                 resultFuture.map {
                   case JobResult(_, result) =>
-                    ctx.complete(jobReport ++ resultToTable(result))
+                    result match {
+                      case s: Stream[_] =>
```

Comment:
Have you tested this?    GET /jobs/<jobId> could be used to retrieve a streams multiple times, want to make sure this works.